packages/foundry/src/create.ts

```diff
@@ -12,6 +14,7 @@ const TemplateGenerators: Record<TemplateId, (outputDirectory: string, options:
         const scope = options["packageScope"];
 
         await cloneProjectTemplate(outputDirectory, `${BaseRepositoryAddress}/host-application`);
+        await cloneProjectTemplate(outputDirectory, `${BaseRepositoryAddress}/vscode-config`);
```

Comment:
Maybe we should rename "cloneProjectTemplate" for "cloneTemplate" ? Since it doesn't only clone projects now.

Reply:
True 👍